support/kueue.go

```diff
@@ -65,7 +64,25 @@ func CreateKueueClusterQueue(t Test, clusterQueueSpec kueuev1beta1.ClusterQueueS
 	return clusterQueue
 }
 
-func CreateKueueLocalQueue(t Test, namespace, clusterQueueName string) *kueuev1beta1.LocalQueue {
+func LocalQueueWithNamespace(namespace string) LocalQueueOption {
+	return func(lq *kueuev1beta1.LocalQueue) {
+		lq.ObjectMeta.Namespace = namespace
+	}
+}
+
+func LocalQueuewithClusterQueue(clusterQueueName string) LocalQueueOption {
+	return func(lq *kueuev1beta1.LocalQueue) {
+		lq.Spec.ClusterQueue = kueuev1beta1.ClusterQueueReference(clusterQueueName)
+	}
+}
+
+func LocalQueuewithAnnotations(annotations map[string]string) LocalQueueOption {
+	return func(lq *kueuev1beta1.LocalQueue) {
+		lq.ObjectMeta.Annotations = annotations
+	}
+}
+
+func CreateKueueLocalQueue(t Test, options ...LocalQueueOption) *kueuev1beta1.LocalQueue {
```

Comment:
That should be `options ...Option[*kueuev1beta1.LocalQueue]`.

Then a struct implementing that option should be created like `DefaultLocalQueue`.

You can find an example here: https://github.com/astefanutti/kube-schedulers/blob/395ea23620b3b6635790a81b7d79288acdebff0f/test/support/options.go#L28